rllab/misc/tensorboard_output.py

```diff
@@ -1,9 +1,14 @@
-import os
+import shutil
+from os.path import dirname, abspath
 
+import google.protobuf.json_format as json_format
```

Comment:
Please ensure all deps are in environment.yml

Even if a package happens to be accessibly because some other dependency uses it (transitive dependency), we need to make sure that environment.yml lists all dependencies in _our_ code.